src/main/java/com/patchworkmc/event/EventConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.patchworkmc.event;
+
+public final class EventConstants {
```

Comment:
Not exactly constants (`getRegisterInstanceDesc`). Maybe `EventDescriptors`?

Reply:
Ehh, it's technically two constants with a wrapper because that would be ugly otherwise. I can change it but beware of 30 million classes holding only public static final members.